### PR TITLE
Countdown Timer

### PR DESCRIPTION
**Pull request by tyseto2**
Screenshot 2023-02-21 210029

### CODE REVIEW

File: client/components/countdown.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+    deadline.setSeconds(deadline.getSeconds() + 10);
+    return deadline;
+  };
+
+  useEffect(() => {
+    clearTimer(getDeadTime());
+  }, []);
+
+  return <h2 className={styles["wecehacks-countdown-nums"]}>{timer}</h2>;
```
Comment (exrlla) [inline client/components/countdown.jsx:61]: beautiful amazing wonderful lfg

File: client/pages/wecehacks.jsx [hunk 2]
```diff
@@ -84,19 +89,42 @@ export default function WECEHacks() {
...
+              </p>
+              <p
+                style={{
+                  fontSize: "1.5em",
+                  fontFamily: "IBM Plex Mono, monospace",
+                  color: "#ecd5ed",
+                }}
+              >
+                until WECEHacks begins.
+              </p>
```
Comment (exrlla) [inline client/pages/wecehacks.jsx:126]: I added this to just add some styling to your countdown

File: client/styles/WECEHacks.module.css [hunk 3]
```diff
@@ -135,18 +137,61 @@
...
+
+@media screen and (max-width: 600px) {
+  .wecehacks-countdown-nums {
+    font-size: 1.5em;
+    letter-spacing: 1rem;
+  }
+
+  .wecehacks-countdown-header {
+    word-spacing: 3rem;
+  }
```
Comment (exrlla) [inline client/styles/WECEHacks.module.css:180]: mobile view css

**Review by exrlla — COMMENTED** [review]
Looks all good w me! I changed some CSS for the image pictures too that shouldn't affect your counter!

**Review by 6sannasia — COMMENTED** [review]
I noticed in the original screenshot the countdown is closer to the top, but now it's after the blurb. Countdowns are more eye-catching, so it'd attract more attention in between WECE Hacks and its date. Was there a reason for moving it down?

**Comment by exrlla** [comment]
> I noticed in the original screenshot the countdown is closer to the top, but now it's after the blurb. Countdowns are more eye-catching, so it'd attract more attention in between WECE Hacks and its date. Was there a reason for moving it down?

I figured the date should go beforehand, and the time should go right after, but other than that, there's no other reason! @tyseto2 feel free to go in and check it out and see how you'd change it? Or if you want to keep it like that?

**Comment by tyseto2** [comment]
I think that putting the date first makes sense but I like having the timer before the text blurb. I rearranged it so that you can see the logo, wecehacks text, and date first now and the timer is right under.
image
image

**Review by 6sannasia — APPROVED** [review]
Looks good!! :DD

**Pull request merged**